lib/ansible/modules/cloud/scaleway/scaleway_lb_facts.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: scaleway_lb_facts
```

Comment:
I follow the convention presents here: https://docs.ansible.com/ansible/2.8/modules/list_of_cloud_modules.html 574 modules end up with facts I will follow this convention.

Reply:
Just because most modules do it wrong doesn't mean new modules are allowed to do that as well :) ansible_facts are supposed to be host-specific, and information on cloud providers usually is not.